src/Nox.Localization/DbContext/NoxLocalizationDbContext.cs

```diff
@@ -1,69 +1,66 @@
 using Microsoft.EntityFrameworkCore;
-using Nox.Localization.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Nox.Abstractions;
 using Nox.Solution;
 using Nox.Types.EntityFramework.Abstractions;
+using Nox.Types.EntityFramework.Enums;
 
-namespace Nox.Localization.DbContext;
+namespace Nox.Localization;
 
 public class NoxLocalizationDbContext: Microsoft.EntityFrameworkCore.DbContext
 {
     private const string LocalizationSchema = "l10n";
 
+    private readonly INoxDatabaseProvider? _databaseProvider;
     private readonly NoxSolution _noxSolution;
 
-    private readonly INoxDatabaseProvider _dbProvider;
-
-    private readonly INoxClientAssemblyProvider _clientAssemblyProvider;
-
     // Schema: 'i10n'
     public DbSet<Translation> Translations { get; set; } = default!;
-
+    
     public NoxLocalizationDbContext(
         NoxSolution noxSolution,
-        INoxDatabaseProvider databaseProvider,
-        INoxClientAssemblyProvider clientAssemblyProvider
-    ) 
+        IEnumerable<INoxDatabaseProvider> databaseProviders
+    )
     {
-        _dbProvider = databaseProvider;
-        _clientAssemblyProvider = clientAssemblyProvider;
         _noxSolution = noxSolution;
+        _databaseProvider = databaseProviders.Single(p => p.StoreType == NoxDataStoreType.EntityStore);
     } 
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     {
         base.OnConfiguring(optionsBuilder);
-
         var appName = _noxSolution.Name;
 
         var dbServer = _noxSolution.Infrastructure?.Persistence.DatabaseServer;
-
+    
         if(dbServer is not null)
         {
-            _dbProvider.ConfigureDbContext(optionsBuilder, appName, dbServer);
+            _databaseProvider!.ConfigureDbContext(optionsBuilder, appName, dbServer);
         }
-
-        //optionsBuilder.MigrationsAssembly("ImportExportLocalization");
-
+        
     }
 
     protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
do you want to delete this method, everything is commented, and we are not going by this path?